libs/csm/extensions/csm_ext.py

```diff
@@ -107,9 +112,66 @@ def destroy_prep_deploy_cluster(self, expect_fail=None):
             self.log.info("[Start] Sleep %s", self.update_seconds)
             time.sleep(self.update_seconds)
             self.log.info("[End] Sleep %s", self.update_seconds)
-         
+
             self.log.info("Cleanup: Check cluster status")
             resp = self.ha_obj.poll_cluster_status(self.master)
             assert resp[0], resp[1]
             self.log.info("Cleanup: Cluster status checked successfully")
         self.log.info("[End] Redeploy the setup...")
+
+    def delete_data_pod(self):
+        """
+        Degrade cluster safely by deleting a data pod
```

Comment:
can u mention specifically that we are deleting last data pod

Reply:
updated